language-api/src/main/java/de/jplag/options/LanguageOptions.java

```diff
@@ -56,7 +56,7 @@ protected <T> LanguageOption<T> createDefaultOption(OptionType<T> type, String n
      * @return The new option
      */
     protected <T> LanguageOption<T> createOption(OptionType<T> type, String name, String description) {
-        LanguageOption<T> option = new DefaultLanguageOption<>(type, name, description);
+        LanguageOption<T> option = new DefaultLanguageOption<>(type, description, name);
```

Comment:
Why did you change the order? The parameters of the function are name and description